include/xnnpack.h

```diff
@@ -1826,6 +1826,8 @@ enum xnn_status xnn_create_runtime(
 struct xnn_external_value {
   uint32_t id;
   void* data;
+  size_t num_dims;
+  const size_t* dim;
```

Comment:
From API point of view, these are the only changes that would be noticed from building the subgraph? Even then, it would be fine to leave `num_dims` and `dim` to be empty/null?